# joint angles
float32 arm0_joint0
float32 arm0_joint1
float32 arm1_joint0
float32 arm1_joint1

# angular velocities
float32 arm0_omega0
float32 arm0_omega1
float32 arm1_omega0
float32 arm1_omega1

# angular accelerations
float32 arm0_accel0
float32 arm0_accel1
float32 arm1_accel0
float32 arm1_accel1

# determine use for data
bool is_goal
bool success
